core/template.js

```diff
@@ -13,10 +13,18 @@
 		rePlaceholder = /{([^}]+)}/g,
 		reEscapableChars = /([\\'])/g,
 		reNewLine = /\n/g,
-		reCarriageReturn = /\r/g;
+		reCarriageReturn = /\r/g,
+		isIndirectEvalSupported = true;
```

Comment:
we should probably expose this flag somehow to tests can run with and without the flag.